scss/components/status-label.scss

```diff
@@ -48,133 +48,119 @@ $fd-status-label-icon-size: fd-space(4) !default;
   --fd-status-label-color: var(--fd-color-text-2);
   --fd-status-label-icon-background-color: var(--fd-color-text-2);
 
-    //LOCAL VARS (set all themeable properties, always include !default)
```

Comment:
Unfortunately, this file shows a bunch of changes -- mostly because of fixing the indentation level for many lines.  The essence of the change was to remove the two mixins that weren't being used and then changing the seven modifier classes to be sibling selectors rather than standalone classes.